pumpkin/Cargo.toml

```diff
@@ -4,6 +4,11 @@ version = "0.1.0-dev"
 description = "Empowering everyone to host fast and efficient Minecraft servers."
 edition = "2021"
 
+[package.metadata.winresource]
+FileDescription = "Pumpkin"
```

Comment:
Please use the description above i think it was something like `package.description`

Reply:
It would be the package description by default, but I changed it to "Pumpkin", because it appeard as "Empowering everyone to host fast and efficient Minecraft servers." in the task manager. I looked through other application metadata and all just used "FileDescription" as the name. ~~_Got to love Windows XD_~~. There is also an issue about it https://github.com/BenjaminRi/winresource/issues/6

Reply:
This is just hilarious, Okay then i can understand why keeping it Pumpkin, But i think it would be great to make a comment above explaining that this is basically not a description but the Name